### ISSUE TITLE
AMM addPool docs outdated by minimum requirement

### ISSUE THREAD

**Issue by dckc** [Agoric/documentation#691]
https://agoric.com/documentation/zoe/guide/contracts/constantProductAMM.html#creating-a-new-pool  is out-of-date w.r.t. https://github.com/Agoric/agoric-sdk/issues/5032

the tests have details:  https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js#L42-L77

one person in #dev figured it out: https://gist.github.com/alexesDev/1bd4efb5e0528f3f0f5174ef883e9af7

**Comment by dckc** [comment]
result of pairing tonight: https://github.com/Agoric/documentation/tree/dc-amm-snippet [ecbb5cc](https://github.com/Agoric/documentation/commit/ecbb5cce3a66f59320804b98b1274f24bd38c42f)

**Comment by Tyrosine22** [comment]
Yup... looks like an active issue, although it does seem fairly low priority. I'll take care of it whenever I revise the contract docs. If somebody wants this taken care of sooner, pls upgrade the priority of this item.
